composeApp/src/commonMain/kotlin/org/ooni/probe/ui/results/ResultsScreen.kt

```diff
@@ -85,33 +87,35 @@ fun ResultsScreen(
                     }
                 }
             },
+            colors = CustomColors.topAppBar(),
         )
 
-        Row(
-            verticalAlignment = Alignment.CenterVertically,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(horizontal = 16.dp)
-                .padding(bottom = 8.dp),
-        ) {
-            Text(
-                stringResource(Res.string.TestResults_Overview_FilterTests),
-                modifier = Modifier.weight(2f),
-            )
+        Surface(color = MaterialTheme.colorScheme.primaryContainer) {
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 16.dp),
+            ) {
+                Text(
+                    stringResource(Res.string.TestResults_Overview_FilterTests),
+                    modifier = Modifier.weight(2f),
+                )
 
-            DescriptorFilter(
-                current = state.filter.descriptor,
-                list = state.descriptorFilters,
-                onFilterChanged = { onEvent(ResultsViewModel.Event.DescriptorFilterChanged(it)) },
-                modifier = Modifier.weight(3f).padding(horizontal = 4.dp),
-            )
+                DescriptorFilter(
+                    current = state.filter.descriptor,
+                    list = state.descriptorFilters,
+                    onFilterChanged = { onEvent(ResultsViewModel.Event.DescriptorFilterChanged(it)) },
+                    modifier = Modifier.weight(3f).padding(horizontal = 4.dp),
+                )
```

Comment:
Title for installed descriptors goes to multiple lines which is ok but then there is no way to tell where one end and the other starts.

![Screenshot_20241003_194835](https://github.com/user-attachments/assets/d25f87e2-a514-43ad-b5a7-bdb21e20b827)